elements/rh-cta/rh-cta.css

```diff
@@ -209,43 +212,43 @@ svg {
  *****************************************************************************/
 
 :host(:not([variant])) {
-  --rh-cta-background-color: transparent;
-  --rh-cta-border-color: transparent;
-  --rh-cta-color: var(--rh-color-interactive-blue, #0066cc);
-  --rh-cta-hover-background-color: transparent;
-  --rh-cta-hover-border-color: transparent;
-  --rh-cta-hover-inner-border-color: transparent;
-  --rh-cta-hover-color: var(--rh-color-interactive-blue-darkest, #004080);
-  --rh-cta-hover-text-decoration: none;
-  --rh-cta-focus-background-color: transparent;
+  --_background-color: var(--rh-cta-background-color,transparent);
+  --_border-color: var(--rh-cta-border-color, transparent);
+  --_color: var(--rh-cta-color, var(--rh-color-interactive-blue, #0066cc));
+  --_hover-background-color: var(--rh-cta-hover-background-colo, transparent);
```

Comment:
--_hover-background-color: var(--rh-cta-hover-background-color, transparent);

Reply:
all that copy and paste was bound to be a typo ;)